src/azure-cli/HISTORY.rst

```diff
@@ -60,6 +60,10 @@ Release History
 * Update azure-mgmt-storage version to 7.0.0 to use api version 2019-06-01
 * Add new parameters `--enable-delete-retention` and `--delete-retention-days` to support managing delete retention policy for storage account blob-service-properties.
 
+**SQL**
```

Comment:
Please remove the release notes at the top of history

Reply:
I don't understand, what release notes should be removed? Can you please explain?